library/core/src/slice/memchr.rs

```diff
@@ -46,11 +46,17 @@ pub fn memchr(x: u8, text: &[u8]) -> Option<usize> {
     // - body, scan by 2 words at a time
     // - the last remaining part, < 2 word size
     let len = text.len();
-    let ptr = text.as_ptr();
     let usize_bytes = mem::size_of::<usize>();
```

Comment:
You could make this a const.

Reply:
I could, but I think it would just make the diff bigger without any added value.